nemo_run/core/execution/lepton.py

```diff
@@ -55,6 +55,7 @@ class LeptonExecutor(Executor):
     lepton_job_dir: str = field(init=False, default="")
     custom_spec: dict[str, Any] = field(default_factory=dict)
     pre_launch_commands: list[str] = field(default_factory=list)  # Custom commands before launch
+    reservation_config: Optional[dict[str, Any]] = None
```

Comment:
Should this be of type `str` instead of `dict`? Looking at the [Lepton API](https://github.com/leptonai/leptonai/blob/main/leptonai/api/v1/types/deployment.py#L16) it appears it just uses string.